src/components/periodic-report/dto/due-report.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { SecuredProperty } from '../../../common';
+import { IPeriodicReport, PeriodicReport } from './periodic-report.dto';
+
+@ObjectType()
+export class DueReport {
+  @Field(() => IPeriodicReport, { nullable: true })
+  readonly current: PeriodicReport | null;
+
+  @Field(() => IPeriodicReport, { nullable: true })
+  readonly next: PeriodicReport | null;
+}
+
+@ObjectType({
+  description: SecuredProperty.descriptionFor('due report'),
+})
+export class SecuredDueReport extends SecuredProperty(DueReport) {}
```

Comment:
Probably doesn't make sense since this is just a holder/grouping of individually secured props.